src/Interfaces/LazySet.jl

```diff
@@ -60,1559 +21,5 @@ The function
 is optional because there is a fallback implementation relying on `σ`.
 However, for unbounded sets (which includes most lazy set types) this fallback
 cannot be used and an explicit method must be implemented.
-
-The subtypes of `LazySet` (including abstract interfaces):
-
-```jldoctest; setup = :(using LazySets: subtypes)
-julia> subtypes(LazySet, false)
-14-element Vector{Any}:
- AbstractAffineMap
```

Comment:
I think this docstring should stay with `LazySet`.

Reply:
I'm not sure about that, `subtypes(LazySet, true)` seems to be recursive and would also print all subtypes of `ConvexSet`, which are already listed in `ConvexSet` docstring, this way it synthetically lists `LazySet` direct subtypes, `ConvexSet` and (one day) the non-convex sets in the package.

Reply:
Yes, remove them from `ConvexSet` of course.